docs/references/api/aggregate_functions.rst

```diff
@@ -175,15 +167,16 @@ You will then get the summed amount, along with the embedded customer resource:
     }
   ]
 
-.. note::
- The previous example uses a has-one association to demonstrate this functionality, but you may also use has-many associations as grouping columns, although there are few obvious use cases for this.
+The previous example uses a "to-one" relationship, but this can be done on "to-many" relationships as well (although there are few obvious use cases).
 
-Using Aggregate Functions Within the Context of an Embedded Resource
---------------------------------------------------------------------
+This also works in a similar way for :ref:`spread embedded resources <spread_embed>`.
+For example, ``select=amount.sum(),...customers(name)`` would sum the ``amount`` grouped by the ``name`` column.
```

Comment:
I think the whole "Grouping with Columns from a Spreaded Resource" can be summarized under this section, since it behaves in a similar way to the non-spread counterpart.

Reply:
Sounds good. The above was a really long subheading.